oauthenticator/tests/googlegroups/fakeadmin_groups.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Hey @missingcharacter! I don't feel very comfortable having these huge jsons copy-pasted from google admin-sdk. I believe the tests were ok before.

I propose to drop these last 2 commits and then I'll merge the PR. We can always refine the tests or the code later, if there are any issues, as we're not breaking any existing functionality. What do you think @missingcharacter ?

Reply:
awesome, I'm totally fine with dropping the last 2 PRs :)

Reply:
Done! :)